packages/core/http/core-http-server/src/router/route_validator.ts

```diff
@@ -154,10 +154,28 @@ export type RouteValidatorFullConfigRequest<P, Q, B> = RouteValidatorConfig<P, Q
  * )
  * ...
  * ```
+ *
+ *  * @example
+ * ```ts
+ * {
+ *    200: {
+ *       body: schema.stream()
+ *       bodyContentType: 'application/octet-stream'
+ *    }
+ * }
+ *
  * @public
  */
 export interface RouteValidatorFullConfigResponse {
-  [statusCode: number]: { body: LazyValidator };
+  [statusCode: number]: {
+    /**
+     * A string representing the mime type of the response body.
+     * @public
+     */
+    bodyContentType?: string;
+    /** @public */
+    body: LazyValidator;
```

Comment:
NIT: I don't think we need the `@public` annotation on fields (or at least we're not doing it in other public types)?